libcloud/compute/drivers/gce.py

```diff
@@ -159,6 +166,16 @@ def request_aggregated_items(self, api_name):
             self.gce_params = params
             response = self.request(request_path, method='GET').object
             if 'items' in response:
+                if zone:
+                    # Special case when we are handling pagination for a
+                    # specific zone
+                    items = response['items']
+                    response['items'] = {}
+                    response['items'] = {
+                        'zones/%s' % (zone): {
```

Comment:
In aggregated request response, this key will contain the zone name (e.g. ``zones/europe-west1-b``), but the actual key is not used anywhere so the name is irrelevant and could really contain any value (it's just the structure of the actual value that matters).